cg/cli/workflow/mutant/base.py

```diff
@@ -100,3 +98,37 @@ def start_available(context: click.Context, dry_run: bool = False):
             exit_code = EXIT_FAIL
     if exit_code:
         raise click.Abort
+
+
+@mutant.command("store-available")
+@DRY_RUN
+@click.pass_context
+def store_available(context: click.Context, dry_run: bool) -> None:
+    """Run QC checks and store bundles for all finished analyses in Housekeeper."""
+
+    analysis_api: MutantAnalysisAPI = context.obj.meta_apis["analysis_api"]
+
+    exit_code: int = EXIT_SUCCESS
+
+    analysis_api.run_qc_and_fail_analyses(dry_run=dry_run)
+
+    for case_obj in analysis_api.get_cases_to_store():
+        LOG.info(f"Storing deliverables for {case_obj.internal_id}")
+        try:
+            context.invoke(store, case_id=case_obj.internal_id, dry_run=dry_run)
```

Comment:
I agree but this might be out of scope. The function `store` has also a lot of logic (which is not ideal), so this change may involve refactoring of the analysis CLI

Reply:
Is it correct now? Not sure of how to use the store command directly. 🤔